doc/website/getting-started/installation.md

```diff
@@ -27,7 +22,7 @@ make install
 ```
 
 If you would like to use our Cyclone DDS Gateway you have to install Cyclone DDS first, see
-[https://github.com/eclipse-cyclonedds/cyclonedds](https://github.com/eclipse-cyclonedds/cyclonedds).
+[here](https://github.com/eclipse-cyclonedds/cyclonedds).
```

Comment:
why not just `... to install [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) first.`

Reply:
Of course, fixed.